source/module_cell/read_pp_upf100.cpp

```diff
@@ -282,28 +287,24 @@ void Pseudopot_upf::read_pseudo_local(std::ifstream &ifs)
 
 void Pseudopot_upf::read_pseudo_nl(std::ifstream &ifs)
 {
-//	int nb, mb, n, ir, idum, ldum, lp, i, ikk;
-	int nb, mb, ir, idum;
+    //	int nb, mb, n, ir, idum, ldum, lp, i, ikk;
+    int nb, mb, ir, idum, ldum;
```

Comment:
initialize variable one for each line, then initialize the variable with 0

Reply:
Done.